### PR TITLE
Ellipsis/New Roles column for smaller window

### PR DESCRIPTION
**Pull request by rayykim**
## Summary

- ellipsis for the longer names in smaller screen
- made a new column for role tags
- intermediary width fix

## Screenshots (if appropriate)

screen shot 2016-08-23 at 10 53 22 am

### CODE REVIEW

**Comment by rayykim** [comment]
@indirectlylit @DXCanas warning: accidentally made a big PR since it wasn't merged earlier this week

**Comment by indirectlylit** [comment]
Are changes in this PR that you didn't intend to be? Or do you mean that there are more commits than you'd like?

**Comment by rayykim** [comment]
@indirectlylit more commits than i'd like

**Comment by indirectlylit** [comment]
okay. We can show you how to squash them if you'd like

**Comment by DXCanas** [comment]
@indirectlylit We're failing some travis errors.

**Comment by DXCanas** [comment]
@rtibbles Care to weigh in on this travis check error?

**Comment by rtibbles** [comment]
Can be ignored. It's a caching issue on Travis that is now fixed on master.

File: kolibri/plugins/management/assets/src/vue/user-page/index.vue [hunk 1]
```diff
@@ -52,13 +53,18 @@
...
 
-            <!-- Logic for role tags -->
+          <!-- Logic for role tags -->
+          <td class="table-cell table-role">
             <span class="user-role" v-for="role in user.roles">
               {{role.kind | capitalize}}
             </span>
-          </th>
+          </td>
+<!--           </th> -->
```
Comment (indirectlylit) [inline kolibri/plugins/management/assets/src/vue/user-page/index.vue]: make sure to delete any comments like these

File: kolibri/plugins/management/assets/src/vue/user-page/index.vue [hunk 2]
```diff
@@ -263,7 +270,7 @@
     padding-left: 1em
     padding-right: 1em
     border-radius: 40px
-    margin-left: 20px
+    // margin-left: 20px
```
Comment (indirectlylit) [inline kolibri/plugins/management/assets/src/vue/user-page/index.vue]: delete commented out code

**Comment by indirectlylit** [comment]
This looks a bit odd when only learners are visible - there's basically an empty column which looks like a bug:

image

Recommend that if we're using the "Admin" badges as before and are just trying to align them, we simply remove the "Role" header text from this column.

**Comment by rayykim** [comment]
@indirectlylit fixed changes from your comments. let me know if there's anything else that needs to be updated

**Comment by indirectlylit** [comment]
thanks

**Comment by DXCanas** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;9HPigt3G&#x2F;536-updates-to-user-management-page"> updates to user management page</a>

**Pull request merged**